powergraph.py

```diff
@@ -25,7 +25,7 @@
 NREAD = 10
 INFINITY = False
 STORE = False
-
+FEEDBACK = False
```

Comment:
I suggest to split the patch in two: one for the run-all-patch and another for this one. Also, ensure you run pylint on all files.